hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/otus-go-prof/hw01_hello_otus
```

Рекомендую указать полный путь до модуля
```suggestion
module github.com/ak7sky/otus-go-prof/hw01_hello_otus
```